src/Kernel/Process.class.st

```diff
@@ -604,6 +604,31 @@ Process >> signalException: anException [
 	oldList ifNotNil: [self resume]
 ]
 
+{ #category : 'testing' }
+Process >> state [
```

Comment:
I wonder if #status would be a better name instead of #state. I am not insist , just thinking. 
State may have a different meaning like an entire copy of a process stack. Status sounds less ambiguous.